notebooks/README.md

```diff
@@ -247,6 +246,8 @@
 - [Image-to-Video synthesis with AnimateAnyone and OpenVINO](./animate-anyone/animate-anyone.ipynb)
 - [Lightweight image generation with aMUSEd and OpenVINO](./amused-lightweight-text-to-image/amused-lightweight-text-to-image.ipynb)
 - [Part Segmentation of 3D Point Clouds with OpenVINO™](./3D-segmentation-point-clouds/3D-segmentation-point-clouds.ipynb)
+- - [Create a RAG system using OpenVINO and LangChain](./llm-rag-langchain/llm-rag-langchain.ipynb)
```

Comment:
please revert, it is automatically generated and will be updated in next commit after merge